File: bubble_sort/bubbleSort.java
import java.util.Scanner;
class BubbleSort 
{ 
    void Sort(int arr[]) 
    { 
        int n = arr.length; 
        for (int i = 0; i < n-1; i++) 
            for (int j = 0; j < n-i-1; j++) 
                if (arr[j] > arr[j+1]) 
                { 
                    int temp = arr[j]; 
                    arr[j] = arr[j+1]; 
                    arr[j+1] = temp; 
                } 
    } 
  
    
    void printArray(int arr[]) 
    { 
        int n = arr.length; 
        for (int i=0; i<n; ++i) 
            System.out.print(arr[i] + " "); 
        System.out.println(); 
    } 
  
     
    public static void main(String args[]) 
    { 
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the length of the array:");
		int length = sc.nextInt();
		int [] arr = new int[length];
		System.out.println("Enter the elements of the array:");

		for(int i=0; i<length; i++ ) 
		{
			arr[i] = sc.nextInt();
		}
		BubbleSort ob = new BubbleSort();
        ob.Sort(arr); 
        System.out.println("Sorted array"); 
        ob.printArray(arr); 
    } 
} 

File: bubble_sort/bubbleSort.py
def bubbleSort(arr):
	n = len(arr)


	for i in range(n):


		for j in range(0, n-i-1):


			if arr[j] > arr[j+1] :
				arr[j], arr[j+1] = arr[j+1], arr[j]


arr = [7, 8, 6, 4, 5, 3, 2, 1, 9, 10]

bubbleSort(arr)

print ("Sorted array is:")
for i in range(len(arr)):
	print ("%d" %arr[i]);
